src/DiffOp/Diff/Diff.php

```diff
@@ -12,6 +12,8 @@
  * Base class for diffs. Diffs are collections of DiffOp objects,
  * and are themselves DiffOp objects as well.
  *
+ * @SuppressWarnings(PHPMD.ExcessiveClassComplexity)
+ *
```

Comment:
Do you think we could get rid of this easily? If so, I'd like to have a task for that on phabricator. If not, I'd prefer to increase the respective limit to just enough for this class to pass.